{
  "errors": {
    "internalServerException": "مشکلی بوجود آمده است"
  }
}
